docs/source/introduction.rst

```diff
@@ -1,23 +1,36 @@
 Introduction to Glow
 ====================
 
-Glow aims to simplify genomic workflows at scale. The best way to accomplish this goal is to take a system that
-has already been proven to work and adapt it to fit into the genomics ecosystem.
+Genomics data has been doubling every seven months globally. It has now reached a scale where genomics has 
+become a big data problem. However, most of the tools for working with genomics data are built to work on 
+single nodes and will no longer scale. Furthermore, it has become extremely challenging for scientists to 
+manage storage of massive volumes of public data.
 
-Apache Spark and in particular `Spark SQL <https://spark.apache.org/sql/>`_, its module for working with
-structured data, is used at organizations across industries with datasets at the petabyte scale and
-beyond. Glow seamlessly integrates common bioinformatics tools into this big data ecosystem so that you can be productive immediately.
+Glow solves these problems by bridging bioinformatics into the big data ecosystem. The library enables bioinformaticians 
+and computational biologists to leverage best practices used by data engineers and data scientists across industry.
```

Comment:
industries?

Reply:
across industry is used as a term for across all industries